cfgov/ask_cfpb/tests/models/test_pages.py

```diff
@@ -737,3 +742,37 @@ def test_get_nonexistent_reusable_text_snippet(self):
         self.assertEqual(
             get_reusable_text_snippet('Nonexistent Snippet'),
             None)
+
+    def test_category_meta_image_empty(self):
+        category_page = self.create_category_page(ask_category=self.category)
+        self.assertIsNone(category_page.meta_image)
+
+    def test_category_meta_image(self):
+        category = mommy.make(Category, category_image=self.social_image)
+        category_page = self.create_category_page(ask_category=category)
+        self.assertEqual(category_page.meta_image, self.social_image)
+
+    def test_answer_meta_image_empty(self):
+        self.assertIsNone(self.page2.meta_image)
```

Comment:
This kind of test is a little bit hard to follow as a reader would have to figure out what `self.page2` is supposed to be.